src/frontend/src/pages/ChangeRequestDetailPage/ChangeRequestDetailsView.tsx

```diff
@@ -187,7 +291,7 @@ const ChangeRequestDetailsView: React.FC<ChangeRequestDetailsProps> = ({
     </div>
   );
 
-  const actionDropdown = changeRequest.accepted ? implementCrDropdown : unreviewedActionsDropdown;
+  const actionDropdown = changeRequest.accepted ? implementCrDropdown : unreviewedActionsDropdown();
```

Comment:
Change implementCrDropdown to use the ActionsMenu component

Reply:
@RChandler234 Right, using ActionsMenu component will be efficient!

Although, just realized that ActionsMenu component has its title hard-coded as `Actions`  in ActionsMenu.tsx. Would it be better to define a separate component for `Implement Change Request` title, or is there another way we could do this job efficiently? 

For now, I think it will be better to leave it as it is.

Reply:
just add a prop called title to the component that defaults to be actions and can be overridden

Reply:
^^